pytest_dependency.py

```diff
@@ -6,9 +6,11 @@
 """
 
 import pytest
+# from blessings import Terminal
```

Comment:
It does not really make sense to me to keep the out-commented import here.  I'd rather remove that entirely.  Same remark applies to the other out-commented stuff below.

In principle, you could put that in a try statement and fall back not to use blessing on ImportError.  On the other hand, I'm not sure if it is really a good idea to meddle with pytest's coloring schema in the first place, that's why I'd rather drop that.

Reply:
You're right. I commented the code, just because the blessing dependence wasn't in setup.py, and I did't want to alter so much the project. 

The colors is not really a big deal specially not to alter the default pytest's schema, so let's forget it. Thanks for pointing out.